chsdi/views/search.py

```diff
@@ -167,6 +168,13 @@ def staging_filter(staging):
         self.sphinx.SetLimits(0, layerLimit)
         self.sphinx.SetRankingMode(sphinxapi.SPH_RANK_WORDCOUNT)
         self.sphinx.SetSortMode(sphinxapi.SPH_SORT_EXTENDED, '@weight DESC')
+        # Weights defaults to 1
+        self.sphinx.SetFieldWeights({
+            '@title': 4,
+            '@detail': 2,
+            '@layer': 1
```

Comment:
layer is the technical name, that's for us only and if you have the technical name well you usually only have one result. but I can change it.